b9lab-content/2-what-is-cosmos/04-atom-staking.md

```diff
@@ -0,0 +1,184 @@
+# Getting ATOM and Staking
+
+<!--
+
+This content could be linked to from https://cosmos.network/learn/get-atom as a step-by-step.
+
+-->
+
+The [Cosmos Hub](https://hub.cosmos.network) is the first among equals of Cosmos blockchains. The ATOM is the native token of the Cosmos Hub, also called the staking token. You need some ATOM to be able to transact on the Cosmos Hub, or to participate in the proof-of-stake consensus. Because of its usefulness, there are markets that determine the ATOM price in common currencies.
+
+So let's get our feet wet. We are going to:
+
+1. Get some ATOM.
+2. Participate in the consensus, thereby earning yield on our ATOMs while sleeping, or studying further.
+
+We cover all the main concepts in the next chapter. For now, you will get through with a surface-level understanding.
+
+The simple step of _get some ATOM_ hides multiple steps:
+
+1. Install a wallet application, and configure it safely. We will use [Keplr](https://keplr.app/).
+2. Buy some ATOM on an exchange, and transfer them to your wallet.
+
+Let's get started.
+
+## Set up Keplr
+
+First the easiest part, open Google Chrome on your computer and go to the [Keplr extension](https://chrome.google.com/webstore/detail/keplr/dmkamcknogkgcdfhhbddcghachkejeap) page. Alternatively, you could search by yourself for Keplr in the [Chrome extensions store](https://chrome.google.com/webstore/search/Keplr). Indeed, it is always good practice to be a bit suspicious of links to wallets as sometimes fraudulent ones are published in order to steal from you. Good thing, Keplr is mentioned, and linked, [here](https://cosmos.network/ecosystem/wallets), lending it some credibility.
+
+When it is installed, it should be available in your list of extensions when you click on the icon in the top-right corner of Chrome:
+
+![Keplr seen as an extension](./images/keplr-as-extension.png)
+
+You can go ahead and pin it if you want.
+
+<HighlightBox type="warn">
+
+When you own ATOM with Keplr, **you are in charge**. For real. There is no one you can call because _you forgot your password_, _your hard-disk crashed_, or _you sent to the wrong address_.
+
+In the jargon, Keplr is a non-custodial wallet.
+
+</HighlightBox>
+
+Now, with the Keplr extension installed and the above warning, let's create a new account. Click on Keplr and the following page should open:
+
+![Keplr create menu](./images/keplr-create-menu.png)
+
+Click _Create new account_. The page that opens offers you a mnemonic and asks you for a password. Because **you are in charge**, it is important to understand the following points:
```

Comment:
```suggestion
Click _Create new account_. The page that opens offers you a mnemonic (a secret list of words) and asks you for a password. Because **you are in charge**, it is important to understand the following points:
```